packages/apps/plugins/plugin-debug/src/DebugPlugin.tsx

```diff
@@ -44,6 +44,7 @@ export const DebugPlugin = (): PluginDefinition<DebugPluginProvides> => {
         .prop(settings.values.$debug!, 'debug', LocalStorageStore.bool)
         .prop(settings.values.$devtools!, 'devtools', LocalStorageStore.bool);
 
+      // TODO(burdon): Remove hacky dependency on global variable?
```

Comment:
Do you want to expose this in the UI? Otherwise, not sure how else to get playwright access.

Reply:
I didn't realize this was for playwright.
What do you think about passing some option in the the client API, where the client calls you back with an "root access" API or additional control plane. This is fine if it's a one-off, but it tends not to be -- e.g., Dima has at least 2 different ways of poking the client from the console.

Reply:
Yeah, that sounds reasonable. FWIW there's other things within the app that use this pattern as well of exposing things on window for playwright. Not that I think that makes it better, but just that it's not just client that needs a solution. I'd been thinking that I'd like a way to expose them only when running for playwright.